HelloQuad/HelloQuad.cs

```diff
@@ -0,0 +1,191 @@
+using AtomicEngine;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        Application.Run<HelloQuad>(args);
+    }
+}
+
+public class HelloQuad : AppDelegate
+{
+    // Scene reference kept here so it won't be collected by the GC
+    Scene scene;
+    Camera camera;
+    Graphics graphics;
+    Viewport viewport;
+
+    Texture2D texture;
+    VertexBuffer vertexBuffer;
+
+    public override unsafe void Start()
+    {
+        // We get the variables we are going to use in this example
+        Renderer renderer = GetSubsystem<Renderer>();
+        graphics = GetSubsystem<Graphics>();
+        viewport = renderer.GetViewport(0);
+
+        // We create a new Scene
+        scene = new Scene();
+        // The Octree should be added to the root scene node (mandatory?)
+        scene.CreateComponent<Octree>();
```

Comment:
I've seen some basic Urho examples and all of them add the Octree even when clearly not necessary, so my guess is it's necessary for some reason, I could be wrong though, but I couldn't find a way to make it work without it, maybe we could ask on the Urho forums.